Jenkinsfile

```diff
@@ -2,7 +2,7 @@ def baseConfig = {
     owner = 'ksql'
     slackChannel = '#ksql-alerts'
     ksql_db_version = "0.10.0"  // next version to be released
```

Comment:
Wanna bump this to 0.11.0 in this PR as well, if convenient? (I can also do this in a follow-up instead.)